.goreleaser.yml

```diff
@@ -89,6 +90,12 @@ release:
     - Read the [documentation](https://autobrr.com)
     - Join our [Discord server](https://discord.gg/WQ2eUycxyT)
 
+source:
+  enabled: true
+  name_template: '{{ .ProjectName }}_{{ .Version }}_src'
```

Comment:
I'd change this to `{{ .ProjectName }}_{{ .Version }}_web_dist` so it's clear it's only web and only the dist dir.